src/Symfony/Bundle/TwigBundle/Tests/DependencyInjection/TwigExtensionTest.php

```diff
@@ -30,8 +30,6 @@ public function testLoadEmptyConfiguration()
         $container->loadFromExtension('twig', array());
         $this->compileContainer($container);
 
-        $this->assertEquals('Twig_Environment', $container->getParameter('twig.class'), '->load() loads the twig.xml file');
```

Comment:
I guess this tests needs to be replaced by testing whether the `twig` service is defined to keep that test